src/main/java/bio/terra/common/db/JdbcProperties.java

```diff
@@ -0,0 +1,46 @@
+package bio.terra.common.db;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "terra.common.db")
```

Comment:
JdbcProperties is supposed to be the common class that would be used directly by each service, as well as a super class of `StairwayJdbcProperties`
StairwayJdbcProperties overried the prefix is okay, but not sure if that is the best way of doing that. Or we want a "common super class" on top of that, and is not a `ConfigurationProperties`

Reply:
Yes, maybe I'll rename this `BaseJdbcProperties` and put a comment that clients should extend the class and put `@ConfigurationProperties` with the appropriate prefix.